### ISSUE TITLE
Crash when entering Aether.

### ISSUE THREAD

**Issue by Jerry996666** [gildedgames/aether-issues#497]
Steps to Crash: Place Aether portal and enter, when aether loads, client crashes. Unload to log back into server until player data is wiped. Works fine on server on same pc, moved server to second pc on same network, crash happens.

Java: Java 7u45

Mods on client:
Latest Aether/gilded games util, Mine & Blade Battlegear 2, Everlasting Tools/SpaceCore, BiblioCraft, CustomNPCs, ForgeMultipart, Immibis core/microblocks, SGCraft, The Mists of Riov (Modded to remove npcs and worlds), Xerxes Weapons, Sword Art Online HUD, BetterFoliage, Custom Main Menu, GokiStats, HungerStrike, The World R:Craft Mod (http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/2377095-the-world-r-craft-hack-in-minecraft), Minecraft Loader, MineMenu, Mystcraft, Player API, Secret Rooms Mod, SGCraft, Help Fixer, Tinkers Defense, Xaeros Minimap, TConstruct, Shaders Mod Core, Mystcraft (Tested without as well), Optifine (also tested without), liteloader, legacyjavafixer, Hunger Strike, Hide Names, Decocraft, Chisel2, Cosmetic Armor, CoFHCore 3.0.0B9-40, CodeChickenLib 1.1.1.88, Damage Indicators Mod 3.2.0, and Smart Moving. All found on The World R:Craft modpack on Technic

Mods on server:
Latest Aether/gilded games util, Mine & Blade Battlegear 2, Everlasting Tools/SpaceCore, BiblioCraft, CustomNPCs, ForgeMultipart, Immibis core/microblocks, SGCraft, The Mists of Riov (Modded to remove npcs and worlds), Xerxes Weapons, GokiStats, HungerStrike, The World R:Craft Mod (http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/2377095-the-world-r-craft-hack-in-minecraft), MineMenu, Mystcraft, Player API, Secret Rooms Mod, SGCraft, Help Fixer, Tinkers Defense, TConstruct, Mystcraft (Tested without as well), Hunger Strike, Hide Names, Decocraft, Chisel2, Cosmetic Armor, CoFHCore 3.0.0B9-40, CodeChickenLib 1.1.1.88, and Smart Moving.

Crash log: 
---- Minecraft Crash Report ----
// Don't be sad. I'll do better next time, I promise!

Time: 5/19/15 9:36 PM
Description: Ticking entity

java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Byte
	at net.minecraft.entity.DataWatcher.func_75683_a(SourceFile:83)
	at net.aetherteam.aether.entities.util.EntitySaddleMount.isSaddled(EntitySaddleMount.java:88)
	at net.aetherteam.aether.entities.util.EntitySaddleMount.func_70071_h_(EntitySaddleMount.java:36)
	at net.aetherteam.aether.entities.mounts.EntityFlyingCow.func_70071_h_(EntityFlyingCow.java:80)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1994)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.entity.DataWatcher.func_75683_a(SourceFile:83)
	at net.aetherteam.aether.entities.util.EntitySaddleMount.isSaddled(EntitySaddleMount.java:88)
	at net.aetherteam.aether.entities.util.EntitySaddleMount.func_70071_h_(EntitySaddleMount.java:36)
	at net.aetherteam.aether.entities.mounts.EntityFlyingCow.func_70071_h_(EntityFlyingCow.java:80)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)

-- Entity being ticked --
Details:
	Entity Type: flyingCow (net.aetherteam.aether.entities.mounts.EntityFlyingCow)
	Entity ID: 443
	Entity Name: Flying Cow
	Entity's Exact location: -267.00, 38.00, 1926.00
	Entity's Block location: World: (-267,38,1926), Chunk: (at 5,2,6 in -17,120; contains blocks -272,0,1920 to -257,255,1935), Region: (-1,3; contains chunks -32,96 to -1,127, blocks -512,0,1536 to -1,255,2047)
	Entity's Momentum: 0.00, -0.08, 0.00
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1887)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['AzureZhen'/428, l='MpServer', x=-246.50, y=74.62, z=1912.50]]
	Chunk stats: MultiplayerChunkCache: 150, 150
	Level seed: 0
	Level generator: ID 01 - flat, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 183359644 game time, 813471 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 8 total; [EntitySheepuff['Sheepuff'/442, l='MpServer', x=-268.50, y=39.00, z=1921.59], EntityFlyingCow['Flying Cow'/443, l='MpServer', x=-267.00, y=38.00, z=1926.00], EntitySheepuff['Sheepuff'/445, l='MpServer', x=-251.66, y=75.00, z=1879.25], EntityItem['item.tile.whiteRose'/446, l='MpServer', x=-284.66, y=34.13, z=1934.41], EntityCarrionSprout['Carrion Sprout'/470, l='MpServer', x=-232.00, y=52.00, z=1918.00], EntityCarrionSprout['Carrion Sprout'/471, l='MpServer', x=-232.00, y=51.00, z=1920.00], EntityCarrionSprout['Carrion Sprout'/465, l='MpServer', x=-277.00, y=77.00, z=1889.00], EntityClientPlayerMP['AzureZhen'/428, l='MpServer', x=-246.50, y=74.62, z=1912.50]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2432)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:909)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.7.0_75, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2267678136 bytes (2162 MB) / 2659188736 bytes (2536 MB) up to 8589934592 bytes (8192 MB)
	Mod Pack: Unknown / None
	LiteLoader Mods: 2 loaded mod(s)
          - WorldEditCUI version 1.7.10_00
          - VoxelLib version 2.3.1
	LaunchWrapper: 49 active transformer(s)
          - Transformer: cpw.mods.fml.common.asm.transformers.PatchingTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.MarkerTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.SideTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.EventSubscriptionTransformer
          - Transformer: mods.battlegear2.coremod.transformers.EntityPlayerTransformer
          - Transformer: mods.battlegear2.coremod.transformers.ModelBipedTransformer
          - Transformer: mods.battlegear2.coremod.transformers.NetClientHandlerTransformer
          - Transformer: mods.battlegear2.coremod.transformers.NetServerHandlerTransformer
          - Transformer: mods.battlegear2.coremod.transformers.PlayerControllerMPTransformer
          - Transformer: mods.battlegear2.coremod.transformers.ItemRendererTransformer
          - Transformer: mods.battlegear2.coremod.transformers.MinecraftTransformer
          - Transformer: mods.battlegear2.coremod.transformers.ItemStackTransformer
          - Transformer: mods.battlegear2.coremod.transformers.ItemInWorldTransformer
          - Transformer: mods.battlegear2.coremod.transformers.EntityAIControlledByPlayerTransformer
          - Transformer: mods.battlegear2.coremod.transformers.EntityOtherPlayerMPTransformer
          - Transformer: com.spacechase0.minecraft.spacecore.asm.block.BlockDestructionTransformer
          - Transformer: com.spacechase0.minecraft.spacecore.asm.network.PacketInterceptTransformer
          - Transformer: com.spacechase0.minecraft.spacecore.asm.render.TextureInterceptTransformer
          - Transformer: com.spacechase0.minecraft.spacecore.asm.mcp.LoaderModInfoTransformer
          - Transformer: com.spacechase0.minecraft.spacecore.asm.mod.ModInitTransformer
          - Transformer: com.spacechase0.minecraft.spacecore.asm.config.GuiFactoryTransformer
          - Transformer: mods.betterfoliage.loader.BetterFoliageTransformer
          - Transformer: cofh.asm.TransformerCore
          - Transformer: cofh.asm.PCCASMTransformer
          - Transformer: mods.immibis.microblocks.coremod.MicroblockSupporterTransformer
          - Transformer: mods.immibis.microblocks.crossmod.MicroblocksBCTransformer
          - Transformer: com.creativemd.minecraftloader.MinecraftTransformer
          - Transformer: api.player.forge.PlayerAPITransformer
          - Transformer: net.smart.core.SmartCoreTransformer
          - Transformer: com.mumfrey.liteloader.transformers.event.EventProxyTransformer
          - Transformer: com.mumfrey.liteloader.launch.LiteLoaderTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.CrashReportTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.DeobfuscationTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.AccessTransformer
          - Transformer: net.minecraftforge.transformers.ForgeAccessTransformer
          - Transformer: mods.battlegear2.coremod.transformers.BattlegearAccessTransformer
          - Transformer: cofh.asm.PCCAccessTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.ModAccessTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.ItemStackTransformer
          - Transformer: com.gildedgames.util.threadedlighting.asm.TLTransformer
          - Transformer: shadersmodcore.transform.SMCClassTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.TerminalTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.LiteLoaderEventInjectionTransformer
          - Transformer: com.thevoxelbox.common.TimeUpdateTransformer
          - Transformer: com.mumfrey.liteloader.common.transformers.LiteLoaderPacketTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.MinecraftOverlayTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.ModAPITransformer
          - Transformer: mods.immibis.core.impl.MultiInterfaceClassTransformer
          - Transformer: mods.immibis.core.impl.TraitTransformer
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx9216m -XX:MaxPermSize=256m
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.85.1291 Minecraft Forge 10.13.2.1291 51 mods loaded, 51 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	FML{7.10.85.1291} [Forge Mod Loader] (modpack.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	Forge{10.13.2.1291} [Minecraft Forge] (modpack.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	<CoFH ASM>{000} [CoFH ASM Data Initialization] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	ImmibisMicroblocks{59.0.4} [Immibis's Microblocks] (immibis-microblocks-59.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	minecraftloader{0.1} [MinecraftLoader] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	PlayerAPI{1.2} [Player API] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	SmartCore{1.0} [Smart Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	mod_ThreadedLighting{1.7.10-1.0} [Threaded Lighting] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	gilded-games-util{1.7.10-1.1} [Gilded Games Utility] (gilded-games-util.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	battlegear2{1.0.7.0} [Mine & Blade Battlegear 2 - Bullseye] (1.7.10-MB_Battlegear2-Bullseye-1.0.7.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	SC0_SpaceCore{0.7.12} [SpaceCore] ([1.7.10] SpaceCore 0.7.12.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	bspkrsCore{6.15} [bspkrsCore] ([1.7.10]bspkrsCore-universal-6.15.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	DamageIndicatorsMod{3.2.0} [Damage Indicators] ([1.7.10]DamageIndicatorsMod-3.2.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	SC0_EverlastingTools{1.0.3} [Everlasting Tools] ([1.7.2] Everlasting Tools 1.0.3.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	saohud{v1.0} [Sword Art Online HUD] ([1.7.2] Sword Art Online HUD v1.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	aether{Alpha 1.7.10-1.4} [Aether II] (aether.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	BetterFoliage{1.0.3} [Better Foliage] (BetterFoliage-1.0.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	BiblioCraft{1.7.4} [BiblioCraft] (BiblioCraft[v1.7.4][MC1.7.10].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	ForgeMultipart{1.1.0.314} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.0.314-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	chisel{2.3.7.34} [Chisel 2] (Chisel2-2.3.7.34.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	CoFHCore{1.7.10R3.0.0B9} [CoFH Core] (CoFHCore-[1.7.10]3.0.0B9-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	CosmeticArmor{1.0.5} [Cosmetic Armor] (CosmeticArmor-1.7.10-1.0.5.B8-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	CustomMainMenu{1.0} [Custom Main Menu] (CustomMainMenu-MC1.7.10-1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	customnpcs{1.7.10} [CustomNpcs] (CustomNPCs_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	props{1.12} [Decocraft] (Decocraft-v1.12_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	EnndsCore{2.1.1.109-b} [Ennds] (EnndsCore-2.1.1.109-b.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	McMultipart{1.1.0.314} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.0.314-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	gokiStats{1.0.0} [gokiStats] (gokiStats-1.7.2-1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	HelpFixer{1.0.7} [HelpFixer] (HelpFixer-1.0.7.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	hidenames{1.1.7} [Hide Names] (Hide Names-1.7.10-1.1.8.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	hungerstrike{1.7.10.4} [Hunger Strike] (HungerStrike-1.7.10.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	ImmibisCore{59.0.4} [Immibis Core] (immibis-core-59.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	Mantle{1.7.10-0.3.2.jenkins184} [Mantle] (Mantle-1.7.10-0.3.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	MineMenu{1.2.0} [MineMenu] (MineMenu-1.7.10-1.2.0.B44-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	Mystcraft{0.11.1.00} [Mystcraft] (mystcraft-1.7.10-0.11.1.00.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	secretroomsmod{4.7.1} [The SecretRoomsMod] (secretroomsmod-1.7.10-4.7.1.376.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	SGCraft{1.8.2} [Greg's SG Craft] (SGCraft-1.8.2-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	SmartMoving{15.2} [Smart Moving] (SmartMoving-1.7.10-15.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	SmartRender{2.0} [Smart Render] (SmartRender-1.7.10-2.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	TConstruct{1.7.10-1.8.2.build858} [Tinkers' Construct] (TConstruct-1.7.10-1.8.2a.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	RioV{v2.4.0} [The Mists of RioV] (The Mists of Riov-1.7.10-1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	examplemod{1.0} [Example Mod] (The Mists of Riov-1.7.10-1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	theworldrcraft{1.0.0} [The World R:Craft] (The-World-R-Craft-1.7.10 v1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	tinkersdefense{1.0} [Tinkers' Defense] (TinkersDefense-1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	worldedit{6.0-beta-01} [WorldEdit] (worldedit-forge-mc1.7.10-6.0-beta-01.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	XaeroMinimap{1.2.2} [Xaero's Minimap] (xaeros_minimap_v1.2.2_1.7.10_Forge.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	MoT{1.0.0.0} [Minecraft of Tomorrow] (XW-1.7.10-1.0.1.4b.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	SA{Alpha 0.0.0.1} [Seven Arts Mod] (XW-1.7.10-1.0.1.4b.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	XW{1.0.0.0} [Xerxes Weapon Mod] (XW-1.7.10-1.0.1.4b.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	ForgeMicroblock{1.1.0.314} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.0.314-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	Mantle Environment: Environment healthy.
	TConstruct Environment: Environment healthy.
	Launched Version: 1.7.10-Forge10.13.2.1291
	LWJGL: 2.9.1
	OpenGL: AMD Radeon R9 200 Series GL version 4.4.13292 Compatibility Profile Context 14.502.1014.0, ATI Technologies Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)

**Comment by Jerry996666** [comment]
This is my log in server log:

[23:15:48] [Netty IO #2/ERROR] [FML]: NetworkDispatcher exception
java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:1.8.0_45]
	at sun.nio.ch.SocketDispatcher.read(Unknown Source) ~[?:1.8.0_45]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source) ~[?:1.8.0_45]
	at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:1.8.0_45]
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source) ~[?:1.8.0_45]
	at io.netty.buffer.UnpooledUnsafeDirectByteBuf.setBytes(UnpooledUnsafeDirectByteBuf.java:436) ~[UnpooledUnsafeDirectByteBuf.class:?]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:870) ~[AbstractByteBuf.class:?]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:208) ~[NioSocketChannel.class:?]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:87) [AbstractNioByteChannel$NioByteUnsafe.class:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:480) [NioEventLoop.class:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:447) [NioEventLoop.class:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:341) [NioEventLoop.class:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:101) [SingleThreadEventExecutor$2.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_45]
[23:15:48] [Server thread/INFO] [net.minecraft.network.NetHandlerPlayServer]: AzureZhen lost connection: TranslatableComponent{key='disconnect.genericReason', args=[Internal Exception: java.io.IOException: An existing connection was forcibly closed by the remote host], siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null}}
[23:15:48] [Server thread/INFO] [net.minecraft.server.MinecraftServer]: §a[Admin]§r §c[OP] §rAzureZhen left the game

**Comment by xJonL** [comment]
@Jerry996666 I managed to get a client most of the listed mods. Try looking into updating all your mods or contact the modpack owner to update them for you. I had some conflicting enchantment ID that was ID 101 with The Mists of RioV mod and couldn't seem to get Smart Moving to work what so ever. I also had to allocate more memory to my client as it ran out of memory when loading my world. I allocated 2GB it sure did the trick.

https://gist.github.com/91bd6f99ef4e890ed962
